mkosi/config.py

```diff
@@ -2290,6 +2291,17 @@ def parse_ini(path: Path, only_sections: Collection[str] = ()) -> Iterator[tuple
         help="Postinstall script to run outside image",
         compat_names=("FinalizeScript",),
     ),
+    ConfigSetting(
+        dest="postoutput_scripts",
+        long="--postoutput-script",
+        metavar="PATH",
+        name="PostOutputScripts",
+        section="Content",
+        parse=config_make_list_parser(delimiter=",", parse=make_path_parser()),
+        paths=("mkosi.postoutput",),
+        path_default=False,
+        help="Output introspection script to run outside image",
```

Comment:
I'm not sure introspection is the best word to use here, at least alone, since this is supposed to also modify outputs, right?

Reply:
Currently I have it that the `/work/out` (`$OUTPUTDIR`) is mounted read-only, as I would expect that the output shouldn't be modified after it was built, hence only introspect the output.

Although I can change that if its too strict.

Reply:
The thing is that this script might want to add more outputs of its own. SBOM manifest or something could be a good example. So let's keep the output directory writable.

Reply:
And suggestions on what to use for the description, my mind is blanking?

Reply:
How about
```suggestion
        help="Output postprocessing script to run outside image",
```